bigquery_etl/glam/templates/client_probe_counts_extract_v1.sql

```diff
@@ -0,0 +1,47 @@
+-- TODO: Remove deduping when dupes are fixed.
+WITH deduped AS (
+    SELECT
+        *,
+        ROW_NUMBER() OVER(
+            PARTITION BY
+                ping_type,
+                os,
+                app_version,
+                app_build_id,
+                channel,
+                metric,
+                metric_type,
+                key,
+                client_agg_type,
+                agg_type
+            ORDER BY
+                total_users DESC
+        ) AS rank
+    FROM
+        `{{ dataset }}.{{ prefix }}_view_client_probe_counts_v1`
+    WHERE
+        channel IS NOT NULL
+        AND app_version IS NOT NULL
+        AND total_users >= 100
+)
+
+SELECT
+    channel,
```

Comment:
@robhudson I know you had numeric representations for desktop's channel and wondering if you had thoughts on whether they are needed here?

Reply:
Good call out, thanks.

I think for Glean I will leave out the numeric representations. The data is smaller and it also seems like it is in fluctuation, at least with the Fenix and Fennec transition, and having that as constants seems like not such a great idea. So this is good as-is.